src/System.Net.Http/src/System/Net/Http/HttpClientHandler.Net46.cs

```diff
@@ -304,7 +304,9 @@ public IWebProxy Proxy
                 return _proxy;
             }
 
+#if NET46
```

Comment:
I decided to do it anyway so that if and when someone makes a similar pass again (as we paste in more reference source, more are brought in) it's quite clear they don't need to delete this one.